src/lib.rs

```diff
@@ -16,7 +16,7 @@
 
 extern crate bitcoin_hashes;
 extern crate byteorder;
-extern crate miniscript;
+extern crate elements_miniscript as miniscript;
```

Comment:
In 79387fd1541cf6d067641cace4288a31149bbb52:

If we are going to replace `miniscript` with `elements_miniscript` and keep the name, why did the previous commit in this PR rename `miniscirpt` to `elements_miniscirpt` in a bunch of places?

Also all these `extern crate`s should be pub IMO.

Reply:
I changed `elements_miniscript` to `(crate::)miniscript` and replaced `extern crate` with `pub use`.